lib/Service/DisplayNotifyService.php

```diff
@@ -316,6 +316,23 @@ public function notifyByCampaignId($campaignId)
                 INNER JOIN `lkcampaignlayout`
                 ON `lkcampaignlayout`.layoutId = `lklayoutdisplaygroup`.layoutId
              WHERE `lkcampaignlayout`.campaignId = :assignedCampaignId
+            UNION
+            SELECT `schedule_sync`.displayId,
+                schedule.eventId,
+                schedule.fromDt,
+                schedule.toDt,
+                schedule.recurrence_type AS recurrenceType,
+                schedule.recurrence_detail AS recurrenceDetail,
+                schedule.recurrence_range AS recurrenceRange,
+                schedule.recurrenceRepeatsOn,
+                schedule.lastRecurrenceWatermark,
+                schedule.dayPartId
+             FROM `schedule`
+                INNER JOIN `schedule_sync`
+                    ON `schedule_sync`.eventId = `schedule`.eventId
+                INNER JOIN `lkcampaignlayout`
+                    ON `lkcampaignlayout`.layoutId = `schedule_sync`.layoutId
+             WHERE `lkcampaignlayout`.campaignId = :assignedCampaignId
```

Comment:
You'll need to have the date/time clauses from the first part of this SQL union to restrict to active sync events.